Basic-Car-Maintenance/Shared/Odometer/Views/OdometerView.swift

```diff
@@ -109,6 +113,52 @@ struct OdometerView: View {
 }
 
 #Preview {
-    OdometerView(userUID: "")
+    var viewModel = OdometerViewModel(userUID: nil)
```

Comment:
I moved some formatting, as opposed to appending everything individually, you can append them all together as an array